iotfunctions/metadata.py

```diff
@@ -2539,6 +2545,12 @@ def predict(self, df):
         logger.info(msg)
         return result
 
+    def predictWithStddev(self, df):
+        mean, stddev = self.estimator.predict(df[self.features], return_std=True)
+        msg = 'predicted using model %s' % (self.name)
+        logger.info(msg)
```

Comment:
log if only required. Better to not clutter the log file.

Reply:
done